code/datums/jobs/squads.dm

```diff
@@ -443,15 +444,15 @@ GLOBAL_LIST_EMPTY_TYPED(custom_squad_radio_freqs, /datum/squad)
 	LAZYADDASSOCSIMPLE(GLOB.reverseradiochannels, "[freq]", radio_channel_name)
 	new_squad.faction = squad_faction
 	var/key_prefix = lowertext_name[1]
-	if(GLOB.department_radio_keys[key_prefix])
+	if(GLOB.department_radio_keys[key_prefix] || radio_blacklist[key_prefix])
 		for(var/letter in splittext(lowertext_name, ""))
-			if(!GLOB.department_radio_keys[letter])
+			if(! (GLOB.department_radio_keys[letter] || radio_blacklist[letter]))
```

Comment:
You also made it so if we try to set the key to one of the blacklisted keys it'll allow it :(

```suggestion
			if(!(GLOB.department_radio_keys[letter] && !(letter in radio_blacklist))
```
We also want this to be an `&&`, both conditions should be false for us to actually assign it as the prefix.
(Also I will commit violence for random space addition after the `!`, don't)